packages/rum/src/domain/rumEventsCollection/error/errorCollection.ts

```diff
@@ -6,7 +6,9 @@ import {
   ErrorSource,
   formatUnknownError,
   getTimestamp,
+  Observable,
   RawError,
+  startErrorCollection as startRawErrorCollection,
```

Comment:
the naming collision is not ideal.
If we want to settle on different names between rum and core, I'd propose automatic error collection for core, since it how we refer to it on the logs side

Reply:
Ok to rename `core/domain/errorCollection` to `core/domain/automaticErrorCollection`. Should we rename `RawError` to `AutomaticError`? It would be more explicit.